bcda/bcdacli/cli_test.go

```diff
@@ -644,55 +645,49 @@ func (s *CLITestSuite) TestImportCCLFDirectory() {
 	buf := new(bytes.Buffer)
 	s.testApp.Writer = buf
 
-	args := []string{"bcda", "import-cclf-directory", "--directory", "../../shared_files/cclf/archives/valid/"}
+	path, cleanup := testUtils.CopyToTemporaryDirectory(s.T(), "../../shared_files/cclf/archives/valid2/")
+	defer cleanup()
+
+	args := []string{"bcda", "import-cclf-directory", "--directory", path}
 	err := s.testApp.Run(args)
 	assert.Nil(err)
 	assert.Contains(buf.String(), "Completed CCLF import.")
-	assert.Contains(buf.String(), "Successfully imported 6 files.")
+	assert.Contains(buf.String(), "Successfully imported 2 files.")
 	assert.Contains(buf.String(), "Failed to import 0 files.")
 	assert.Contains(buf.String(), "Skipped 1 files.")
 
 	buf.Reset()
 
-	db.Where("aco_cms_id = ?", "A0001").Find(&existngCCLFFiles)
-	for _, cclfFile := range existngCCLFFiles {
+	db.Where("aco_cms_id = ?", targetACO).Find(&existingCCLFFiles)
+	for _, cclfFile := range existingCCLFFiles {
 		err := cclfFile.Delete()
 		assert.Nil(err)
 	}
-
-	testUtils.ResetFiles(s.Suite, "../../shared_files/cclf/archives/valid/")
-
-	// dir has 4 files, but 2 will be ignored because of bad file names.
```

Comment:
Removed because the test above already verifies the "bad file name" check. 

With the introduction of the fileName unique constraint, this test will directly conflict with an equivalent test in cclf_test.go.

Reply:
Makes sense.